tests/test_diaForcedSource.py

```diff
@@ -37,7 +37,9 @@
 
 
 def create_test_dia_objects(n_points, wcs, startPos=100):
-    """Create dummy DIASources or DIAObjects for use in our tests.
+    """Create dummy DIASources or DIAObjects for use in our tests. Adds
+    Three test sources outsize of the ccd area.
+
```

Comment:
Not in the currently changed code, but I notice below that two of the outside sources are given `src['id'] = 10000001`. Is that a bug?

Reply:
Thanks for the find, fixed.